docs/pages/setup/reference/networking.mdx

```diff
@@ -121,33 +121,47 @@ numbers for each service.
 </TabItem>
 <TabItem scope={["cloud"]} label="Teleport Cloud">
 
-Teleport services listen on several ports. This table shows the default port
-numbers for each service. The Teleport Proxy Service is the host available at your
-Teleport Cloud tenant URL, e.g., `mytenant.teleport.sh`.
+Teleport Cloud allocates a different set of ports to each tenant. To see which
```

Comment:
Would it make sense to move this whole section into a partial/include? It's basically the same as in Cloud FAQ.

Reply:
I'm not sure the partial would be used in guides beside these two, so I'd rather wait to see if we need it later before deciding the best way to create something reusable.